packages/deployer/yarn.lock

```diff
@@ -1237,11 +1237,6 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-13.13.32.tgz#f0edd0fb57b3c9f6e64a0b3ddb1e0f729b6f71ce"
   integrity sha512-sPBvDnrwZE1uePhkCEyI/qQlgZM5kePPAhHIFDWNsOrWBFRBOk3LKJYmVCLeLZlL9Ub/FzMJb31OTWCg2F+06g==
 
-"@types/node@^14.14.22":
```

Comment:
Probably was remaining from Arrowana's PR

Reply:
Cool. I wonder if it's worth us upgrading to Node.js v14 at some point? It's the active LTS version at the moment, but v12 still has another year of support

Reply:
Makes sense to me